src/csharp/Grpc.nuspec

```diff
@@ -5,7 +5,7 @@
     <title>gRPC C#</title>
     <summary>C# implementation of gRPC - an RPC library and framework</summary>
     <description>C# implementation of gRPC - an RPC library and framework. See project site for more info.</description>
-    <version>0.5.0</version>
+    <version>0.5.0.1</version>
```

Comment:
But C# is 0.5.0 and I think it's not being consistent within a specific language is crazy. (I don't wanna have 0.5.1 version for some C# packages and 0.5.0 for some other C# packages). 

If the format inconsistency bothers you, the 0.5.0.1 can be marked as hidden on nuget once we publish the next version. So it's just a temporary thing.